src/olympia/discovery/serializers.py

```diff
@@ -0,0 +1,37 @@
+from rest_framework import serializers
+
+from olympia.addons.models import Addon
+from olympia.addons.serializers import AddonSerializer, VersionSerializer
+from olympia.versions.models import Version
+
+
+class DiscoveryVersionSerializer(VersionSerializer):
+    class Meta:
+        fields = ('compatibility', 'files',)
```

Comment:
I only find it convenient if you have something like

```python
foo = (
    'bar',
    'foo',
)
```
But here. Well, I really don't care and we don't have any guidelines yet. By the way, let's have some more guidelines! :laughing: